network-api/networkapi/wagtailpages/pagemodels/buyersguide/consumer_creepometer_page.py

```diff
@@ -28,3 +30,14 @@ class ConsumerCreepometerPage(BasePage):
     translatable_fields = [
         SynchronizedField("year"),
     ]
+
+    @property
+    def bg_page(self):
+        active_locale = Locale.get_active()
+        return BuyersGuidePage.objects.get(locale=active_locale)
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        bg_page = self.get_parent().get_parent().specific()
```

Comment:
@jhonatan-lopes I made the changes but are now getting

```
TypeError at /en/privacynotincluded/articles/annual-consumer-creep-o-meter/

'BuyersGuidePage' object is not callable
```

Did I miss anything 😬 ?

Reply:
I've messed up, it should be `.specific` without the brackets:

```
bg_page = self.get_parent().get_parent().specific
```